LocalSettings.php

```diff
@@ -2155,6 +2155,7 @@
 	'wgRightsIcon' => array(
 		'default' => 'https://meta.miraheze.org/w/resources/assets/licenses/cc-by-sa.png',
 		'incubatorwiki' => 'https://meta.miraheze.org/w/resources/assets/licenses/cc-by-sa.png',
+		'jadtechwiki' => //static.miraheze.org/jadtechwiki/d/d8/CopyrightIcon.png',
```

Comment:
It's done like that in ManageWiki, but here you must use  "//$wmgUploadHostname instead of static.mh